staging/src/k8s.io/client-go/tools/events/eventseries_test.go

```diff
@@ -158,7 +157,11 @@ func TestEventSeriesf(t *testing.T) {
 	}
 	eventBroadcaster := newBroadcaster(&testEvents, 0, map[eventKey]*eventsv1.Event{})
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, "eventTest")
-	eventBroadcaster.StartRecordingToSink(stopCh)
+	// Let it panic if the underlying type is not `*eventBroadcasterImpl`
+	broadcaster := eventBroadcaster.(*eventBroadcasterImpl)
+	// We don't want to call `StartRecordingToSink` here, as we don't really need `refreshExistingEventSeries` in this test
+	// and unexpected events might be created by `refreshExistingEventSeries` during testing.
```

Comment:
Let's change to something like:

// Don't call StartRecordingToSink, as we don't need neither refreshing event
// series nor finishing them in this tests and additional events updated would
// race with our expected ones.